README.md

```diff
@@ -37,11 +37,13 @@ To work around this:
 1. Restart the browser, and you're in business.
 
 ## Release process:
-  1. Manually bump the version number in `src/install.rdf` and `src/update.rdf`.
-  1. Bump the version number in package.json using `npm version patch`. This will generate a git tag, too.
-  1. Zip up a new addon: `gulp build`.
+  1. Bump the package.json version number and create an XPI using one of the following commands:
```

Comment:
Mind rewording to "Use one of the following commands to bump the package.json version number and create an XPI"?